app/controllers/associations.js

```diff
@@ -47,39 +43,30 @@ export default class IndexController extends Controller {
 
   @action
   setActivities(selectedActivities) {
-    this.page = 0;
+    // TODO: Find out and explain why we are serializing the notation instead of the concept id
```

Comment:
I guess this has the benefit of more readable URL's, with the short notations instead of the long uuid's